tests/CSSList/DocumentTest.php

```diff
@@ -85,4 +86,72 @@ public function setContentsReplacesContentsSetInPreviousCall()
 
         self::assertSame($contents2, $this->subject->getContents());
     }
+
+    /**
+     * @return Generator
```

Comment:
Let's be as specific as possible:

```suggestion
     * @return Generator<initialContent: list<…>, …>
```